### PR TITLE
Finishing touches before v1.0.0 release (closes #199) 

### LINKED ISSUE #199: Check the whole documentation
Make the documentation formatted as uniformly as possible
```js
/**
 * @category Something Helpers
 * @summary Capital, period.
 *
 * @description
 * Capital, period.
 *
 * May be multiline.
 *
 * @param {CapitalizedType} nameOfArgument - hyphen, no capital, no period
 * @returns {CapitalizedType} no capital, no period
 *
 * @example
 * // Capital, date formatted like 29 December 2008, colon:
 * var result = nameOfFunction(arguments)
 * //=> String(result), add milliseconds where needed, remove time zone, no period
 */
```

### PR DESCRIPTION
**Pull request by LeshaKoss**

### CODE REVIEW

**Comment by LeshaKoss** [comment]
@kossnocorp before merging, please check eachDay docs https://github.com/date-fns/date-fns/blob/199-check-docs/src/each_day/index.js#L3 and if needed, please help to rewrite summary and description

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This change log follows the format documented in [Keep a CHANGELOG].
 [Semantic Versioning]: http://semver.org/
 [Keep a CHANGELOG]: http://keepachangelog.com/
 
-## [Unreleased]
+## [1.0.0]
```
Comment (kossnocorp) [inline CHANGELOG.md]: Please keep empty Unreleased section and adjust references:

https___raw_githubusercontent_com_date-fns_date-fns_master_changelog_md_

**Comment by kossnocorp** [comment]
Looking great 🏁 . Not sure about `eachDay` docs, let's keep it as is for now.

File: CHANGELOG.md (hunk 1 above)
Comment (LeshaKoss) [inline CHANGELOG.md]: done fa225d13d017dec0b51f0377ce5f5e54a4a626c5

Comment (LeshaKoss) [inline CHANGELOG.md]: Don't forget to add date to CHANGELOG.md when you're going to make a release

**Pull request merged**